app/views/receipt_mailer/receipt_email.html.erb

```diff
@@ -0,0 +1,22 @@
+<p>Hello,</p>
+
+<p>You are receiving this message to confirm the successful submission of the
+electronic version of your <%= @thesis.graduation_month %> <%=
+@thesis.graduation_year %> thesis "<%= @thesis.title %>" to the MIT Libraries.
+</p>
+
+<p>You do not need to reply to this message.</p>
+
+<p>Theses will be added to DSpace after degrees have been awarded and the thesis
```

Comment:
should dspace be a link?

What happens to people who want a text version of their email? (Does ruby autogenerate multipart with a text version from the html?)

Reply:
Text email is not default. We can add it easily. I'll open a ticket though as I'd like to discuss pros and cons outside the context of this PR.

Reply:
https://mitlibraries.atlassian.net/browse/TI-68

Reply:
👍